tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -25,107 +22,264 @@ Create VCH - custom base disk
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - URL without user and password
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
```

Comment:
doesn't need to be done on each case, but it is easier this way and it is essentially free - on one of the latest runs this keyword only took 36 milliseconds to run when there aren't any to cleanup.